config/locales/en.yml

```diff
@@ -3202,13 +3202,13 @@ en:
             <p>Please include the sum of the following:</p>
             <ul class="list--bulleted">
               <li><a href="https://nj.gov/treasury/taxation/pdf/pubs/tgi-ee/git8.pdf" target="_blank" rel="noopener nofollow">Estimated income tax payments</a> you made throughout the year for your New Jersey state taxes</li>
-              <li>
+              <li class="list--no-bullets">
```

Comment:
[pebble/boulder] This new class should not be necessary. The bullet points will be correct for free if the `ul` is nested within the `li` that has the content it should be nested from (the previous `li`)

So the fix should look something like:
```
<li>
  <a href="https://nj.gov/treasury/taxation/pdf/pubs/tgi-ee/git8.pdf" target="_blank" rel="noopener nofollow">Estimated income tax payments</a> you made throughout the year for your New Jersey state taxes
  <ul class="list--bulleted">
    <li><strong>This does not include withholdings in your paycheck</strong></li>
  </ul>
</li>
```

You can do this for this one and the one below, and no need for the CSS change.